### ISSUE TITLE
Node.js v14 Warning: Accessing non-existent property ___ of module exports inside circular dependency

### ISSUE THREAD

**Issue by ledsun** [jonycheung/deadsimple-less-watch-compiler#107]
**Which version are you using (less-watch-compiler --version to find out)?**

```
~ less-watch-compiler --version
1.14.5
```

**Is the issue reproducible after updating to the latest version ( npm update less-watch-compiler)?**

YES.

**Describe the bug**

The following warning is displayed when running less-watch-compiler in Node.js v14.

```
(node:22520) Warning: Accessing non-existent property 'cat' of module exports inside circular dependency
...
```

**To Reproduce**
```
npx less-watch-compiler
```

**Actual behavior**
```
~ npx less-watch-compiler
(node:21737) Warning: Accessing non-existent property 'cat' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:21737) Warning: Accessing non-existent property 'cd' of module exports inside circular dependency
...
```

**Expected behavior**
No warning is displayed.

**Screenshots**
<img width="766" alt="スクリーンショット 2020-04-24 11 51 52" src="https://user-images.githubusercontent.com/1079508/80170057-08a0fe00-8622-11ea-8b5a-ab5e9a35acef.png">

**Desktop (please complete the following information):**
 - OS: [e.g. Mac OSX 10.14.6]

**Additional context**
It seems that this is due to one of the dependencies. 
shelljs/shelljs#991

**Comment by ledsun** [comment]
I have confirmed that this warning does not appear if shelljs 0.8.4 is installed.

**Comment by Misiur** [comment]
Vote for reopen
